src/ccl_mass_conversion.c

```diff
@@ -0,0 +1,112 @@
+#include <stdio.h>
```

Comment:
If you wanted to put some comments on the formulae in this file, that would be nice. I am not going to hold up this PR because of that though.

Reply:
Added some explanation